packages/database/package.json

```diff
@@ -12,7 +12,8 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "test": "run-p test:browser test:node",
+    "test": "yarn --cwd ../.. test:database:emulator",
```

Comment:
The indirection feels a little unnatural to me. I was confused for a moment where test:database:emulator is.
Can you move the test:database:emulator to the database package?

Reply:
Yes, this makes more sense. Done.